doc/src/documentation_style_guide.rst

```diff
@@ -0,0 +1,1019 @@
+some dependencies like ``dvipng`` or ``docbook2x`` are not available.
+
+For Windows 10, however, the Windows Subsystem for Linux can be a possible
+workaround solution, and you can install Ubuntu shell on your Windows system
+after following the tutorial below:
+
+https://github.com/MicrosoftDocs/WSL/blob/live/WSL/install-win10.md
+
+In your command prompt, run ``ubuntu`` to transfer to Linux terminal, and
+follow the Debian/Ubuntu tutorial above to install the dependencies, and then
+you can run ``make html`` to build. (Note that you also have to install
+``make`` via ``apt-get install make``.)
+
+If you want to change the directory in your prompt to your working folder of
+SymPy in the Windows file system, you can prepend ``cd /mnt/`` to your file
+path in Windows, and run in your shell to navigate to the folder. (Also note
+that Linux uses ``/`` instead of ``\`` for file paths.)
+
+This method provides better compatibility than Cygwin or MSYS2 and more
+convenience than a virtual machine if you partially need a Linux environment
+for your workflow, however this method is only viable for Windows 10 64-bit
+users.
+
+2. Build the Documentation
+--------------------------
+
+The documentation can be built by running the ``makefile`` in the ``doc``
+subdirectory.
+
+To start, in your preferred web browser, use the drop down menu and select
+“open file” to navigate into the sympy/doc folder saved on your computer. In
+the doc folder, select the _build folder, then the html folder, and in the html
+folder, open the index.html file.
+
+To build the HTML documentation, run::
+
+  cd doc
+
+  make html
+
+This builds a local version of the documentation in ``doc/_build/html`` in your
+web browser.
+
+Open ``_build/html/index.html``.
+
+3. Make a Contribution
+----------------------
+
+For in-depth instructions on how to contribute to SymPy’s code base including
+coding conventions, creating your environment, picking an issue to fix, and
+opening a pull request, please read our full `Development Workflow
+<https://github.com/sympy/sympy/wiki/Development-workflow>`_ guide.
+
+Best Practices for Writing Documentation
+========================================
+
+Please follow these formatting, style, and tone preferences when writing
+documentation.
+
+Formatting Preferences
+----------------------
+
+**Math vs. Code**
+
+**Math:** text that is surrounded by dollar signs $ _ $ will be rendered as
+LaTeX math. Any text that is meant to appear as LaTeX math should be written as
+$math$. In the HTML version of the docs, MathJax will render the math.
+
+**Example**
+
+::
+
+  The Bessel $J$ function of order $\nu$ is defined to be the function
+  satisfying Bessel’s differential equation.
+
+**Code:** text that should be printed verbatim, such as code, should be
+surrounded by a set of double backticks ``like this``. (WORK IN PROGRESS: change
+single backticks to render as code via issue `#13519
+<https://github.com/sympy/sympy/issues/13519>`_.)
```

Comment:
Are we going to support both single and double backticks for code?

Reply:
Double backticks will always work for code, the behavior of single backticks can be chosen with `default_role`.